fic/resource_eri_router_paired_to_gcp_connection_v1_test.go

```diff
@@ -0,0 +1,211 @@
+package fic
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+
+	"github.com/nttcom/go-fic"
+
+	connections "github.com/nttcom/go-fic/fic/eri/v1/router_paired_to_gcp_connections"
+
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccPairedRouterToGCPConnection_basic(t *testing.T) {
+	var connection connections.Connection
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "fic_eri_router_paired_to_gcp_connection_v1.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:      func() { testAccPreCheckGCPConnection(t) },
+		Providers:     testAccProviders,
+		CheckDestroy:  testAccCheckPairedRouterToGCPConnectionDestroy,
+		IDRefreshName: resourceName,
```

Comment:
複数のリソースを作成しているので、テスト中にrefreshするリソースを明示しています。

Reply:
refreshというのはreadコールバックを呼んでリソースのstateを更新する処理のことです。テストではapply後にrefreshを実行しています。
> When a test is ran, Terraform runs plan, apply, refresh, and then final plan for each TestStep in the TestCase
https://www.terraform.io/docs/extend/best-practices/testing.html#built-in-patterns

refreshするリソースの名前のデフォルト値は`the first non-nil primary resource`と書かれていますが、今回のテストでは複数のリソースを作成しているので明示的にリソース名を指定する必要があります。
> IDRefreshName is the name of the resource to check. This will default to the first non-nil primary resource in the state.
https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk@v1.15.0/helper/resource?tab=doc#TestCase